pages/newsletter/index.tsx

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
 import Head from 'next/head';
 import Link from 'next/link';
+import { InferGetServerSidePropsType } from 'next';
 import Layout from '../../src/components/partials/Layout';
 import gloss from '../../src/utils/glossary.json';
+import { newsletterRoutes, URL_ACTIONS } from '../../src/utils';
 
+const HOST = process.env.HOST;
+const USER_SERVICE_HOST = process.env.USER_SERVICE_HOST;
 export async function getServerSideProps({ query }) {
-  return { props: { returnParams: query } };
+  const ONE_LOGIN_ENABLED = process.env.ONE_LOGIN_ENABLED === 'true';
+  return {
+    props: {
+      returnParams: query,
+      ONE_LOGIN_ENABLED,
```

Comment:
keep same naming convention as other env config?

```suggestion
      oneLoginEnabled: ONE_LOGIN_ENABLED,
```